clevis.go

```diff
@@ -101,23 +98,38 @@ func (p Pin) ToConfig() (Config, error) {
 
 // Decrypt decrypts a clevis bound message. The message format can be either compact or JSON.
 func Decrypt(data []byte) ([]byte, error) {
-	msg, p, err := Parse(data)
+	return jwe.Decrypt(data, "", nil, jwe.WithPostParser(clevisParser{}))
+}
+
+type clevisParser struct{}
```

Comment:
Personally I am not a fan of using empty structs with a single method. Having a regular function and passing it to `jwe.WithPostParser()` would be cleaner as for me. Though upstream thinks other way unfortunately, so let's go with the  upstream way (like you did now).

Reply:
Indeed, we're at their mercy :).  I would have also preferred passing a function.

Reply:
Perhaps we can let them know and see if they'd be willing to make a change?

Reply:
They're way ahead of us: https://github.com/lestrrat-go/jwx/issues/402#issuecomment-877476832

Reply:
@anatol So yeah, I agree with not wanting to use an empty struct. I _don't_ agree with explicit `func()` based parameters. Please use the `interface{}` based approach that I've put in the PR if you really must use a `func()`